seqr/views/utils/dataset_utils.py

```diff
@@ -227,15 +229,17 @@ def match_and_update_search_samples(
     )
     loaded_date = timezone.now()
     get_individual_sample_key = _get_mapped_individual_lookup_key(sample_id_to_individual_id_mapping)
+    sample_project_tuples = [(sample_id, None) for sample_id in sample_ids]
```

Comment:
for search samples we will not be using hacky null projects for tuples, there is no reason why this needs to change for search samples. Update the helper functions so that for search samples we continue to have a dictionary keyed just on sample ID and for RNA samples we are keying by tuples

Reply:
Okay. I will add a tuple type check when it needs to use the sample_id in the key.